doc/source/tune-usage.rst

```diff
@@ -499,6 +499,50 @@ And stopping a trial (``PUT /trials/:id``):
     curl -X PUT http://<address>:<port>/trials/<trial_id>
 
 
+Tune CLI (Experimental)
+-----------------------
+
+``tune`` has a easy-to-use command line interface (CLI) to manage and monitor your experiments on Ray. To do this, verify that you have the ``tabulate`` library installed:
+
+.. code-block:: bash
+
+    $ pip install tabulate
+
+Here are a few examples of command line calls.
+
+- ``tune ls``: List tabular information about trials within an experiment. Add the ``--sort`` flag to sort the output by specific columns.
```

Comment:
I think for examples, it would be better to use the full commands like `tune list_trials`.